src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const males = people.filter(person => (
+    century
+      ? (Math.ceil(person.died / 100) === century && person.sex === 'm')
+      : (person.sex === 'm')
+  ));
+
+  return Math.round(males.map(man => man.died - man.born)
+    .reduce((acc, man2) => acc + man2) / males.length * 100) / 100;
 }
```

Comment:
you may try to destructure man2 and in that case you won't need to use `map` here